direct/common/subsample.py

```diff
@@ -352,6 +377,190 @@ def __load_masks(self, acceleration):
         return output
 
 
+class RadialMaskFunc(BaseMaskFunc):
+    """
+    Follows CIRCUS implementation from
+        "Liu J, Saloner D. 'Accelerated MRI with CIRcular Cartesian UnderSampling (CIRCUS): a variable density Cartesian
+        sampling strategy for compressed sensing and parallel imaging.'"
+    """
```

Comment:
Can be done like this: https://numpydoc.readthedocs.io/en/latest/format.html#references